### DISCUSSION TITLE
Can you disable image resizing?

### DISCUSSION THREAD

**Discussion by mjking** [jpanther/congo#104] [Q&A] [answered]
Is there a way to disable the image resizing with Hugo pipes? I have all of my images on AWS S3 as a remote URL.

I am getting the following error when I try to build the site:
Markdown image error in "posts/---.md": Resource "https://---.s3-us-west-2.amazonaws.com/---.png" not found. Check the path is correct or remove the image from the content.

**Comment by jpanther** [comment, marked as answer]
Thanks for the question. This is a limitation with the theme currently. The next release will change this behaviour so that when an external URL is provided as the image source, there will be no Hugo image processing.

However, if you want to completely disable any theme logic around image processing, you can create a file in your project at `layouts/_default/_markup/render-image.html` which simply contains the following:

```html
<img src="{{ .Destination }}" alt="{{ .Text }}" title="{{ .Title }}" />
```

**Reply by mjking** [reply]
Thanks! That worked great

**Comment by venteto** [comment]
@jpanther could you add a configuration parameter to globally disable Hugo Pipes image processing, regardless of whether the image source is external to the Hugo site, inclusive of images stored in page bundles, so that no template overriding one day breaks with a newer Congo version?
